ingest/sqs/pools/transformer/pool_transformer.go

```diff
@@ -236,6 +236,11 @@ func (pi *poolTransformer) convertPool(
 			if err != nil {
 				return nil, err
 			}
+		} else if cosmWasmPoolModel.IsOrderbook() {
+			err = pi.updateOrderbookInfo(ctx, pool.GetId(), pool.GetAddress(), cosmWasmPoolModel)
```

Comment:
nit: let's create vars from `pool.GetId(), pool.GetAddress()`and reuse them between alloyed and orderbook